wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -98,15 +98,14 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 - **Dashes may be used if the time between the ticks of the desired snapping is 125ms or higher.** As an example, 1/3 dashes would be allowed at 160 BPM and below, whereas 1/4 dashes would be allowed at 120 BPM and below.
 - **Edge dashes must not be used** as they require extremely precise timing which cannot be expected of less-experienced players.
 - **Basic dashes must not be used more than two times between consecutive fruits. If higher-snapped dashes are used, they must be used singularly (not in conjunction with other dashes).**
-- **Higher-snapped dashes of different beat snap must not be used between consecutive fruits.**
 - **At least 250 ms must be left between objects and the start and end of spinners to ensure readability.** As an example, a 1/3 gap would be required at 80 bpm, whereas a 1/2 gap would be required at 120 BPM.
 
 #### Guidelines
 
 - **All distances should be clear on whether they require the player to walk or dash.** This is to ensure that players can easily recognize patterns that require dashing.
+- Basic dashes of different beat snap should not be used between consecutive fruits.
```

Comment:
is there a reason that some of these aren't bold? just looks inconsistent with formatting. i think u should bold the whole thing if there's only one sentence / main point (this is what other RCs do)

same applies to other parts of this article

Reply:
uh i guess i could go through, they were like this before tho so idek